src/styles/main.scss

```diff
@@ -0,0 +1,157 @@
+@import 'colors';
+@import 'measures';
+
+.container {
+  padding: $paddingV-card $paddingH-card;
+  display: grid;
+  grid-template-columns: repeat(4, $width-card);
+  gap: $row-gap $column-gap;
+  justify-content: center;
+  box-sizing: border-box;
+
+  @media (max-width: 1023px) {
+    grid-template-columns: repeat(3, $width-card);
+  }
+
+  @media (max-width: 768px) {
+    grid-template-columns: repeat(2, $width-card);
+  }
+
+  @media (max-width: 488px) {
+    grid-template-columns: repeat(1, $width-card);
+  }
+}
+
+.card {
+  box-sizing: border-box;
+  width: $width-card;
+  height: 408px;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), you should avoid using fixed height for containers. It's better to use padding and let the content define the height.